packages/react-server/package.json

```diff
@@ -39,8 +39,8 @@
     "winston": "2.2.0"
   },
   "peerDependencies": {
-    "react": "~0.14.2",
-    "react-dom": "~0.14.2",
+    "react": "^15.1.0",
```

Comment:
Is it possible to target `~0.14.2 || ^15.1.0`? That will let users still on react 0.14 catch up